### ISSUE TITLE
Can not deploy on remote server.

### ISSUE THREAD

**Issue by phuc1012000** [TanayB11/sql-injection-demo#5]
This demo works fine on my local machine.
But when I deploy it on server for a demo, it failed with localhost:3000/search denial.
I suspect the problem is related to CORS but do not know how to fix.

**Comment by TanayB11** [comment]
Check `server/app.js` and modify the following code

```javascript
app.use(cors({
  origin: 'your_ip_here'
}))
```

**Comment by phuc1012000** [comment]
It not seems to work. I have change it as you mentioned. 
I think the problem is related with AWS security and fireware so I litterally allow access to any port, but it still fail.
I changed the search call from localhost:3000 to 0.0.0.0 or public and private API but all fail.

I changed the all kind of CORS policy in ExpressJS that show in the document but it not work either.

It 's interesting that when I access publicIP:8080 of my server and also open my local expressjs. Somehow they connect which should be impossible.
